### DISCUSSION TITLE
Installing library with master branch don't have `build-all.sh` ?

### DISCUSSION THREAD

**Discussion by anjopater** [Jobbatical/secrets#11] [Q&A]
Trying to install the library and talking with Miguel we need to run both `Install.sh`  and also `build-install.sh` to generate the secrets and the file  `Install.sh` is in the commit [55923](https://github.com/Jobbatical/secrets/commit/559231296a55d261907c420cbc24f1081ae5be53) and not in master branch so we need to do `checkout` the specific commit to run and generate,  any reason for it or we need add it to the current docs?

@Param-Harrison

**Comment by Param-Harrison** [comment]
@kanolato can help find answer for this man. I don't have much familiarity with the secrets project. But @pedrojimenezp and @jorcalle11 have run this in their windows machine before. You can check with them on how they cracked it.
